packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -74,17 +76,27 @@ class DependencyExtractionWebpackPlugin {
 		return request;
 	}
 
+	serialize( asset ) {
+		if ( this.options.outputFormat === 'php' ) {
+			return `<?php return unserialize('${ phpSerialize( asset ) }');`;
```

Comment:
Why the serialization? Couldn't this just use `json_decode( '${ JSON.stringify( asset ) }', true )`? Way simpler and also reduces the risk of serialization being exploited.

Reply:
Yes, ideally it's something like that as discussed in the comments:

```php
<?php
return array( "dependencies" => [ "a", "b" ], "version" => "abc"  );
```

Reply:
I think I found something promising with d0a983f.